src/asRendition.tsx

```diff
@@ -56,10 +56,12 @@ const styledSystemProps = Object.keys(propTypes);
 const filterStyledSystemProps = (passthroughProps: string[]) => (
 	Base: React.ComponentType<{ ref: any }>,
 ) => {
+	const diff = new Set(difference(styledSystemProps, passthroughProps));
+	const shouldIncludeProp = (prop: string) => !diff.has(prop);
```

Comment:
Sorry that I missed this :(
```suggestion
	const shouldIncludeProp = (_value: unknown, prop: string) => !diff.has(prop);
```